addons/mail/static/src/js/services/mail_document_thread_manager.js

```diff
@@ -213,13 +228,22 @@ MailManager.include({
      * @param {StorageEvent} ev
      */
     _onStorage: function (ev) {
-        if (ev.key === this.DOCUMENT_THREAD_STATE_KEY) {
-            var state = this.call('local_storage', 'getItem', this.DOCUMENT_THREAD_STATE_KEY);
-            this._updateDocumentThreadWindows(state);
+        var value = JSON.parse(ev.newValue);
+        if (ev.key.indexOf(this.DOCUMENT_THREAD_STATE_KEY) !== -1) {
+            if (value.tabId === this._tabId) {
+                return;
+            }
+            // key format: DOCUMENT_THREAD_STATE_KEY + '/' + threadId
+            var documentThreadId = ev.key.substring(this.DOCUMENT_THREAD_STATE_KEY.length+1);
+            var param = {};
```

Comment:
One line assignation ?
```suggestion
            var param = { documentThreadId : value.state };
```